### PR TITLE
Small updates to dash.html

### PR DESCRIPTION
**Pull request by tnyeanderson**
##### Summary

* Improves documentation based on #7221 #6913
* Clarifies that `dash.html` is provided more as a courtesy or an experiment than a core part of netdata. Suggest netdata cloud for users that require more functionality
* Styling improvements! Sticky hostnames so you can see where you are no matter how far you've scrolled!
* Some code cleanup (formatting and whitespace, mostly)

##### Component Name

* dash.html

##### Additional Information

The update process is manual. Back up your current dash.html file, then copy the example file again and recreate any of the changes you had made when setting up dash.html the first time.

##### A Question

One of the things I like about dash.html is that it has only one component: the file itself. All the CSS, JS, and HTML are in the same file. This makes it portable as it simply queries the netdata API, independent of the file location.

However this approach does come with disadvantages: namely the update process.

If different components were stored in different files (perhaps in a dedicated directory), then user configuration could be stored independently and CSS/JS would update transparently to the user upon a netdata upgrade.

However, this does make it seem more "integrated" into the core of netdata, and users might come to expect the netdata team to support it. This was not my original intent, I simply wanted others to have access to my experiment. It is still not my intent, but I wanted to float the idea to the team and get your thoughts.

I figure that netdata wants to promote their cloud platform, which I also encourage. I understand that these changes could mean additional responsibility and possibly less traffic to the cloud platform, so it may not be best from a business perspective.

If the netdata team wants to make this change, I am happy to do so and submit another PR for it, I just want to get your input moving forward with full information.

### CODE REVIEW

**Comment by tnyeanderson** [comment]
This is ready for review, sorry for the extra commits!

**Comment by jacekkolasa** [comment]
@tnyeanderson thanks for you notes! You're right, it would make sense to keep things separated, to allow smooth upgrades. To avoid concerns about being it "officially supported" part, we could put it into some specific folder (`community`?). @cakrit any thoughts from the product perspective?

We'll quite soon merge the new Dashboard (which resides now in separate repository https://github.com/netdata/dashboard) and I would personally wait until we finish the migration.

Anyway, thanks for the maintenance work! :) Sticky is a good idea. Can i just ask if you change the indents back to spacebars? We have agreed on a [common linting rules](https://github.com/netdata/eslint-config-netdata/blob/master/.eslintrc.js), but i haven't put them yet to `netdata` repository (i plan to do that after switching to new dashboard)

**Comment by tnyeanderson** [comment]
Okay the tabs should be (4) spaces now, sorry about that! I like the idea of a "community" folder. Other options could be:

* A separate "community" repository (I like this idea as it allows for these kinds of projects to be easier to maintain in the future)
* A separate repo for this project (don't like this idea as much, if at all)

At the end of the day I want to do whatever the netdata team thinks is best. Either way, this should be ready to merge!

**Comment by joelhans** [comment]
I think we'll end up with a community dashboard repo that builds a very simple gallery site using something like Hugo. Users could then submit PRs with their custom dash HTML/CSS/JS, plus a screenshot and any other instructions (like you have here), to have them added to the gallery.

It's something I'm already exploring because of #7671, but it's admittedly a lower priority for me at the moment. If anyone has ideas, I'm open to anything.

File: web/gui/custom/README.md [hunk 1]
```diff
@@ -74,37 +78,39 @@ First, rename the file so it doesn't get overwritten. For instance, with a webro
 cp /usr/share/netdata/web/dash-example.html /usr/share/netdata/web/dash.html
 ```
 
-Change the following line in `dash.html` to reflect your URLs. The second URL is used if you access your netdata dashboard from a reverse proxy. The reverse proxy URL is optional, if it is not set then both will use the netdata host URL.
+Find and change the following line in `dash.html` to reflect your netdata URLs. The second URL is only used if you access your netdata dashboard through a reverse proxy. The reverse proxy URL is optional; if it is not set then both will use the netdata host URL.
```
Comment (joelhans) [inline web/gui/custom/README.md]: Netdata should always be capitalized.

```suggestion
Find and change the following line in `dash.html` to reflect your Netdata URLs. The second URL is only used if you access your Netdata dashboard through a reverse proxy. The reverse proxy URL is optional; if it is not set then both will use the Netdata host URL.
```

File: web/gui/custom/README.md [hunk 2]
```diff
@@ -65,7 +65,11 @@ header:
 
 ## Dash (Multi-Host Dashboard)
 
-`dash-example.html` is an all-in-one page that automatically fetches graphs from all your hosts. Just add your graphs and charts (or use the defaults) one time using the `dash-*` syntax and it will be automatically replicated for all of your hosts; showing alarms and graphs for all your hosts on **one page!**
+`dash-example.html` is an all-in-one page that automatically fetches graphs from all your hosts. Just add your graphs and charts (or use the defaults) one time using the `dash-*` syntax, and your selections will be automatically replicated for all of your hosts; showing alarms and graphs for all your hosts on **one page!**
+
+__**Dash will only work if you have implemented netdata streaming using `stream.conf`**__
+
+`dash-example.html` was created as an experiment to demonstrate the capabilities of netdata in a multi-host environment. If you desire more features, submit a pull request or check out netdata cloud!
```
Comment (joelhans) [inline web/gui/custom/README.md]:
```suggestion
`dash-example.html` was created as an experiment to demonstrate the capabilities of netdata in a multi-host environment. If you desire more features, submit a pull request or check out Netdata Cloud!
```

**Comment by tnyeanderson** [comment]
@joelhans I think that's a great idea. If someone wants to initialize a `community` repo here I will take care of the leg work getting dash.html in there (with better structure, and a more unique name) and set up a general README and CONTRIBUTE to get it kick-started.

**Comment by joelhans** [comment]
As you've seen in the other thread, @tnyeanderson, we may end up keeping these files within the core repo for now. But, I think we can move ahead with these changes, figure out the right location for custom dashboards, and then figure out the best way to move your dash code to that finalized location.

**Comment by tnyeanderson** [comment]
Sounds good to me, let me know if/how I can help!

**Pull request merged**